gammapy/datasets/map.py

```diff
@@ -2983,6 +2989,8 @@ def to_hdulist(self):
         hdulist = super().to_hdulist()
         exclude_primary = slice(1, None)
 
+        creation = self.meta.creation
+
```

Comment:
Should we not update the creation date at the time of the writing (as made in `map.to_hdulist()`) rather using the date of the its initial creation?